assets/css/style.css

```diff
@@ -0,0 +1,323 @@
+body {
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+
+    font-size: 15px;
+    line-height: 1.6;
+    background-color: #576A7D;
+}
+
+*,
+* :after,
+* :before {
+    box-sizing : border-box
+}
+
+h1, h2, h3, h4, h5, h6 {
+    margin: 0;
+}
+
+.container {
+    width: 100%;
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+.intro {
+    width: 100%;
+    height: 100vh;
+}
+
+
+.header {
+    width: 100%;
+    height: 70px;
+    padding-top: 15px;
+
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1000;
+    background-color: #7F8EA8;
+}
+
+
+.header__inner{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.nav{
+    font-size: 20px;
+    text-transform: uppercase;
+
+    
+}
+
+.nav__link{
+    display: inline-block;
+    vertical-align: top;
+    margin: 0 20px;
+    position: relative;
+
+    color: #fff;
+    text-decoration: none;
+
+    transition: color 0.1s linear;
+}
+
+
+.nav__link:after{
+    content: "";
+    display: block;
+    width: 100%;
+    height: 3px;
+
+
+    background-color: #fff;
+    opacity: 0;
+
+    position: absolute;
+    top:100%;
+    left: 0;
+    z-index: 1;
+
+    transition: opacity 0.1s linear;
+}
+
+.nav__link:hover{
+    color:#fce38a
+}
+
+
+.nav__link:hover:after{
+    opacity: 1;
+}
+
+
+.footer{
+    position: relative; /* Фиксированное положение */
+    left: 0; bottom: 0; 
+    width: 100%;
+    float:left;
+    padding: 1%;

```

Comment:
somewhere u use percents somewhere px, that's not good^ u should be consistent in ur styling, otherwise u will end up with a mess, read about rem it's better to use them, it's also should be applied to ur font-sizes.